packages/edge/src/lib/commands/drivers-util.ts

```diff
@@ -44,8 +43,7 @@ export const buildTableOutput = (tableGenerator: TableGenerator, driver: EdgeDri
 	}
 	return `Basic Information\n${basicInfo}\n\n` +
 		`${deviceIntegrationProfiles}\n\n` +
-		`${fingerprints}\n\n` +
-		summarizedText
+		fingerprints
```

Comment:
In similar places in other files, you left the variable in its wrapper in the string literal even though the newlines were removed.  I could see that keeping all the variables in wrappers might make it easier to skim because they all look the same, but also that it's cleaner to do what you did here.  Is one approach preferable to the other?